### DISCUSSION TITLE
edit product combination after generation in PS 8.1.4

### DISCUSSION THREAD

**Discussion by hogo20** [PrestaShop/PrestaShop#35509] [Ideas]
### The problem

Hi. have a Good time. 
After we create a product combination, there is no way to change or add a feature of that combination.
For example, we can't edit... T-shirt (color :red)
to👉 T-shirt (color :green)
or to 👉T-shirt (color :red, size:xl) 

### The solution

I think it would be nice if there was an option to edit the created combination on the product page. 

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by ElodieOS** [comment]
Hello @hogo20! 
Can you explain a bit of the context, like a use case to make sure I understand the need, please?

**Reply by hogo20** [reply]
Hi, thank you for your attention.🙏
Imagine , you create a product with the following combination and setting the photos and prices. 
product TV(model:sony, size:55inch, Os:Android)
if you want to add new feature to created combination, what to do. 
like TV(... +voltage:230V)
Delete the previous combination and create a new product and set photos and prices again.

Or imagine that you have created a product combination like: box(size:small)
After creation, you need to edit it to box(size:medium). You can't change the attribute in the properties because other products use that feature. Now you have to recreate the whole combinatio!
If the number of products is high, This will be very difficult.

**Reply by ElodieOS** [reply]
Thanks for the explanation, I understand the need better. In terms of UX, the request is completely legitimate, now I don't know if there are any technical restrictions. An opinion @jolelievre?

**Comment by Madmin27** [comment]
While this was easily done in older versions of Prestashop, the inability to edit it in Prestashop 8 and later is very frustrating. I need to change the attributes of the products on my e-commerce site from time to time according to market conditions and our production line. Deleting the combinations that rank in Google search results and creating new ones is not a sensible thing to do. I'm very upset about this.

I could try to overcome this problem by modifying the code myself, but I don't know how deep that would require, so I'm afraid of breaking the system.

Is there anyone who can tell me that editing product combinations in Prestashop 8 and later versions is possible or that it doesn't cause any problems?
